docs/source/tutorials/odc-stac.rst

```diff
@@ -0,0 +1,306 @@
+
+We must specify the URL for the catalog we want to search, along with the desired collection (:code:`io-lulc-annual-v02`) and asset (:code:`data`). 
+The precise items that we need to load will be returned by a query that we run later.
+
+Type the following into the empty cell below the **STAC metadata** heading:
+
+.. code-block:: python
+
+   catalog_url = "https://planetarycomputer.microsoft.com/api/stac/v1/"
+   desired_collections = ["io-lulc-annual-v02"]
+   desired_assets = ["data"]
+
+When you have finished, run the cell by pressing :kbd:`Shift+Enter` on your keyboard.
+
+Connect to catalog and find items
+---------------------------------
+
+We use :class:`pystac-client:pystac_client.Client` to connect to Planetary Computer's STAC catalog.
+We also use planetary-computer's `sign_inplace <https://planetarycomputer.microsoft.com/docs/quickstarts/reading-stac/>`_ modifier to authorise our connection.
+Type the following into the empty cell below the **Connect to catalog and find items** heading:
+
+.. code-block:: python
+
+   stac_client = Client.open(
+      url=catalog_url, 
+      modifier=planetary_computer.sign_inplace,
+   )
+
+When you have finished, run the cell by pressing :kbd:`Shift+Enter` on your keyboard.
+
+Search for items
+^^^^^^^^^^^^^^^^
+
+After setting up the :class:`Client <pystac-client:pystac_client.Client>`, we use :meth:`pystac-client:pystac_client.Client.search` to find items that match our chosen collection, area of interest, and date range.
+Type the following into the empty cell below the **Search for items** heading:
+
+.. code-block:: python
+
+   items = stac_client.search(
+       collections=desired_collections,
+       intersects=desired_aoi_geometry,
+       datetime=desired_date_range,
+   ).item_collection()
+
+   print(f"Found {len(items)} items")
+
+When you have finished, run the cell by pressing :kbd:`Shift+Enter` on your keyboard.
+After running the cell, you should see a printed sentence reporting "Found 7 items"
+
+Troubleshooting
+"""""""""""""""
+
+If the sentence shows a different number of items, try checking whether your :code:`desired_date_range` parameter is correct by printing it:
+
+.. code-block:: python
+
+   print(desired_date_range)
+
+should return :code:`('2017-01-01', '2023-01-01')`.
+If you see a different date range, return to the :ref:`odc-stac-tutorial-date-range` section and ensure your :code:`desired_start_date` and :code:`desired_end_date` values match those given in the instructions.
+
+Load items with odc-stac
+------------------------
+
+After producing a list of items to load, we can use :func:`odc-stac:odc.stac.load` to read the requested assets from the items and return them as xarrays.
+
+Type the following into the empty cell below the **Load items with odc-stac** heading:
+
+.. code-block:: python
+
+   ds = load(
+      items=items,
+      bands=desired_assets,
+      geopolygon=aoi_geometry,
+      crs="utm",
+      resolution=30
```

Comment:
Just noticed that all the other multi-line calls have trailing comma after the last parameter, but this one is missing a trailing comma.

Reply:
Thanks for picking this up. I've added the comma